src/modules/Instabug.ts

```diff
@@ -533,6 +536,19 @@ export const onStateChange = (state?: NavigationStateV5) => {
   }, 1000);
 };
 
+/**
+ * Sets a listener for screen change
+ *  @param navigationRef a refrence of a navigation container
+ *
+ */
+export const setNavigationListener = (
+  navigationRef: NavigationContainerRefWithCurrent<ReactNavigation.RootParamList>,
+) => {
+  navigationRef.addListener('state', () => {
+    onStateChange(navigationRef.getRootState());
+  });
```

Comment:
I think we should also return a function to unregister the listener. That way if for some reason the navigation ref changes the user can stop listening on the old one and listen on the new one instead.

So it could also be used like this with `useEffect`:
```js
useEffect(() => {
  return Instabug.setNavigationListener(navigationRef);
}, [navigationRef]);
```